pkgs/development/python-modules/pook/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, furl
+, isPy3k
+, jsonschema
+, mock
+, nose
+, pytestCheckHook
+, requests
+, xmltodict
+}:
+
+buildPythonPackage rec {
+  pname = "pook";
+  version = "1.0.0";
```

Comment:
1.0.1 was tagged last year: https://github.com/h2non/pook/releases/tag/v1.0.1

Reply:
Ups, I missed that. Thanks.